ff4k-store-sqlite/src/androidUnitTest/kotlin/com/yonatankarp/ff4k/store/SqliteUniqueConstraintTest.android.kt

```
@@ -0,0 +1,124 @@
+            uid = "same-uid",
+            isEnabled = false,
+            description = "Second version",
+        )
+
+        store += feature1
+
+        // When / Then
+        try {
+            store += feature2
+            throw AssertionError("Expected FeatureAlreadyExistsException")
+        } catch (e: FeatureAlreadyExistsException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)